README.md

```diff
@@ -27,4 +36,8 @@ The remote monitoring preconfigured solution illustrates how you can perform end
 
 ### Visual Studio solution:
   * **RemoteMonitoring:** contains both the Dashboard web app, the EventProcessor worker role, and the Simulator worker role.
```

Comment:
Should read: 
**RemoteMonitoring.sln** contains the source code for the whole preconfigured solution, including the solution portal web app, the EventProcessor web job, and the Simulator web job.